modules/rdma-configuring-the-nvidia-network-operator.adoc

```diff
@@ -0,0 +1,230 @@
+kube-ipoib-cni-ds-fsn8g                                       1/1     Running   2          160m
+mofed-rhcos4.16-9b5ddf4c6-ds-ct2h5                            2/2     Running   4          160m
+nic-feature-discovery-ds-dtksz                                1/1     Running   2          160m
+nv-ipam-controller-854585f594-c5jpp                           1/1     Running   2          160m
+nv-ipam-controller-854585f594-xrnp5                           1/1     Running   2          160m
+nv-ipam-node-xqttl                                            1/1     Running   2          160m
+nvidia-network-operator-controller-manager-5798b564cd-5cq99   1/1     Running   2         5d23h
+rdma-shared-dp-ds-p9vvg                                       1/1     Running   0          85m
+----
+
+. You can `rsh` into the `mofed` container to check the status:
++
+[source,terminal]
+----
+$ MOFED_POD=$(oc get pods -n nvidia-network-operator -o name | grep mofed)
+$ oc rsh -n nvidia-network-operator -c mofed-container ${MOFED_POD}
+sh-5.1# ofed_info -s
+OFED-internal-24.07-0.6.1:
+sh-5.1# ibdev2netdev -v
+0000:0d:00.0 mlx5_0 (MT41692 - 900-9D3B4-00EN-EA0) BlueField-3 E-series SuperNIC 400GbE/NDR single port QSFP112, PCIe Gen5.0 x16 FHHL, Crypto Enabled, 16GB DDR5, BMC, Tall Bracket                                                       fw 32.42.1000 port 1 (ACTIVE) ==> ibs2f0 (Up)
+0000:a0:00.0 mlx5_1 (MT41692 - 900-9D3B4-00EN-EA0) BlueField-3 E-series SuperNIC 400GbE/NDR single port QSFP112, PCIe Gen5.0 x16 FHHL, Crypto Enabled, 16GB DDR5, BMC, Tall Bracket                                                       fw 32.42.1000 port 1 (ACTIVE) ==> ens8f0np0 (Up)
+----
+
+. Create a `IPoIBNetwork` custom resource file:
++
+[source,terminal]
+----
+apiVersion: mellanox.com/v1alpha1
+kind: IPoIBNetwork
+metadata:
+  name: example-ipoibnetwork
+spec:
+  ipam: |
+    {
+      "type": "whereabouts",
+      "range": "192.168.6.225/28",
+      "exclude": [
+       "192.168.6.229/30",
+       "192.168.6.236/32"
+      ]
+    }
+  master: ibs2f0
+  networkNamespace: default
+----
+
+. Create the `IPoIBNetwork` resource on the cluster:
++
+[source,terminal]
+----
+$ oc create -f ipoib-network.yaml 
+----
++
+.Example output
+[source,terminal]
+----
+ipoibnetwork.mellanox.com/example-ipoibnetwork created
+----
+
+. Create a `MacvlanNetwork` custom resource file for your other interface: 
++
+[source,terminal]
+----
+apiVersion: mellanox.com/v1alpha1
+kind: MacvlanNetwork
+metadata:
+  name: rdmashared-net
+spec:
+  networkNamespace: default
+  master: ens8f0np0
+  mode: bridge
+  mtu: 1500
+  ipam: '{"type": "whereabouts", "range": "192.168.2.0/24", "gateway": "192.168.2.1"}'
+----
+
+. Create the resource on the cluster:
++
+[source,terminal]
+----
+$ oc create -f macvlan-network.yaml 
+----
++
+.Example output
+[source,terminal]
+----
+macvlannetwork.mellanox.com/rdmashared-net created
+----
+
+
```

Comment:
extra blank line here if you would want to delete it 
```suggestion
```

Reply:
Changed.